### ISSUE TITLE
Add support for opening audio streams via the URL query string

### ISSUE THREAD

**Issue by GreenLunar** [conversejs/converse.js#317]
See jappix/jappix/issues/150 (rejected by admin)

I guess it would attract online radio networks.

Adding a new query string `conversestream=` (more details below)

**Realization**
website?conversemuc=lounge@muc.alternamedia.org&conversestream=http://airtime.classicrocklounge.com:8000/ogg
converse.js users will enjoy the stream while chatting with their favorite radio talk show hosts or their favorite music radio DJs.

```
Radio Name: The Classic Rock Lounge
OGG Stream: http://airtime.classicrocklounge.com:8000/ogg
Home  Page: http://classicrocklounge.com/
Chat  Room: http://www.classicrocklounge.com/chat/
```

**Who might be interested?**
I think that radio show hosts who use chat rooms would be interested in such feature for their web chat room
Example: [Pirate Party Radio](http://rantmedia.ca/piratepartyradio/) (rantmedia.ca) who are currently using their own channel at Freenode.  Note: I will not write it down, there is an XMPP chat room of a Pirate Party website which has over 200 participants, so I think XMPP would be attractive in the future to Rant Media as well.

**Comment by jcbrand** [comment]
It's a neat feature, I like it. Related to #314 which is also about manipulating converse.js via the query string.

Probably best developed as a plugin.

I'm considering moving converse.js out to its own "converse" organisation. In this organisation we can then include plugins for converse.js and also give other people commit rights.

**Comment by GreenLunar** [comment]
> Probably best developed as a plugin.

I think so too. An HTML5 media player plugin.

**Comment by jcbrand** [comment]
I mean a converse.js plugin :)

**Comment by GreenLunar** [comment]
Yes, A converse.js HTML5 media player plugin (or converse.js audio player plugin).

**Comment by jcbrand** [comment]
I've added support for rendering a stream in a MUC via the topic.
That's as far as I'll go with this.

![Image](https://github.com/user-attachments/assets/980ef51f-f940-4894-b14f-dd78ea8d0283)
